### DISCUSSION TITLE
UUID setup

### DISCUSSION THREAD

**Discussion by TheFrozenShoe** [Pryaxis/TShock#2891] [Q&A]
![BF3053A0-D796-4D92-B232-A30A04F4A3EF](https://user-images.githubusercontent.com/122146066/211139444-81c6d129-01d9-4202-8c9b-fe27be50b284.jpeg)
i can no longer use /setup and then the code. console says that im using an automatic UUID login and there is a way to switch it off. i havent been able to find it, how do i get it back to accepting the setup code?

**Comment by sgkoishi** [comment]
They are unrelated.
For `/setup`, it is disabled during startup if any registered users are found. If you expect a new server, delete the whole `tshock` folder which contains the database and all settings and start the server.
For UUID, it's just a warning telling you 'something is enabled by default and is not secure'.
